File: genetic-car-server/src/main/java/fr/genetic/server/web/view/CarScoreView.java
package fr.genetic.server.web.view;

import fr.genetic.server.simulation.Car;

public class CarScoreView {

    public CarView car;
    public float score;

    public static CarScoreView create(Car car) {
        CarScoreView carScoreView = new CarScoreView();
        carScoreView.car = CarView.create(car);
        carScoreView.score = car.getScore();
        return carScoreView;
    }
}

File: genetic-car-starters/genetic-car-starter-python/src/carScoreView.py
class CarScoreView(object):

    def __init__(self, car, score):
        self.car = car
        self.score = score

    def __str__(self):
        return "{{car:{car}, score:{score}}}".format(
            car=str(self.car),
            score=str(self.score)
        )

    def __repr__(self):
        return str(self)
